### ISSUE TITLE
Relative time indication as configuration option

### ISSUE THREAD

**Issue by ThomDietrich** [rejuvenate/lovelace-horizon-card#37]
**Original title:** "Add Clock"

----

@Tontze commented on Dec 4, 2022 in https://github.com/AitorDB/home-assistant-sun-card/issues/155:

> Awesome card ! Would be nice to have current time in this card :)

**Comment by manelrodero** [comment]
Hi Thom,

I have landed on this repository after seeing that the original card had been archived.

I had commented to Aitor some time ago if it would be possible to add the current time on the card in this https://twitter.com/manelrodero/status/1591487879511179264 tweet.

He told me that yes, he would have some work because he was very busy. Now that you have continued the project, could you add something like that?

![image](https://user-images.githubusercontent.com/26098260/230464289-267edca7-59e9-4d11-a809-c2b6b8af0331.png)

PS: Ignore the different font, it was a quick edit to see the concept.

Thanks.

**Comment by ThomDietrich** [comment]
Hey @manelrodero,
I've created the ticket because it existed in the old archived repository. It is certainly up for discussion. A PR is also welcome.

That said, I am not convinced adding the current time is a good idea. You probably have a clock in sight anyhow, or you could add one right next to this card on your dashboard.
Furthermore: Why do you want to know the current time? I have to assume that the actual use case is to get a feeling for the distance to the sown events? In that case it would be more useful to see "Sunset 1:08h ago", do you agree?

**Comment by avataar** [comment]
I agree there is absolutely no need for the card to be yet another clock.
